docs/core/integration-quality-scale/rules/iqs015.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "IQS015 - Entities use DeviceInfo correctly"
+---
+
+IQS015 is as follows:
+
+Entities use DeviceInfo correctly
+
+## Reasoning
+
+Devices are used to group entities into a single device.
```

Comment:
I think we need to have one sentence that introduces the device registry in Home Assistant. Otherwise this sentence is confusing.